packages/components/src/components/hds/icon-tile/index.ts

```diff
@@ -87,7 +81,7 @@ export default class HdsIconTileIndexComponent extends Component {
     } else if (this.args.icon) {
       return this.args.icon;
     } else {
-      return null;
+      return undefined;
```

Comment:
this fixed [this](https://github.com/hashicorp/design-system/pull/2033#issuecomment-2027725005) linting error

Reply:
`undefined` seems to be the correct value here. in the [FlightIcon signature](https://github.com/hashicorp/design-system/blob/main/packages/components/unpublished-development-types/%40hashicorp/ember-flight-icons/flight-icon.d.ts#L8) we define the name argument as `string`, if it exists (otherwise it's `undefined`). also in the component API documentation for the Icon Tile component we state that it must be a string (no mention of allowing null).